src/test/datascience/notebook/remoteNotebookEditor.vscode.test.ts

```diff
@@ -158,6 +159,7 @@ suite('DataScience - VSCode Notebook - (Remote Execution)', function () {
     });
 
     test('Local and Remote kernels are not listed', async function () {
+        await changeShowOnlyOneTypeOfKernel(true);
```

Comment:
Oh and I noticed I hadn't changed this test. It was failing from my experiment submission.